atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,139 @@
+            1. Generates a local embedding for the provided text and UUID.
+            2. Retrieves all cluster medoid embeddings from Weaviate.
+            3. Computes cosine similarity between the input embedding and each medoid.
+            4. Identifies the medoid with the highest similarity score.
+            5. Stores the input embedding in the TEXT collection with a reference to the selected medoid.
+            6. Returns the array of similarity scores.
+    """
+    def newTextPipeline(self, text: str, uuid: str):
+        embedding_id, embedding = generate_embeddings_local(uuid, text)
+        clusters = self.weaviate_client.get_all_embeddings(CollectionNames.CLUSTERCENTER.value)
+        medoids = np.array(entry["vector"] for entry in clusters)
+        similarity_scores = np.array(compute_cosine_similarity(embedding, medoid) for medoid in medoids)
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)